Modules/_cursesmodule.c

```diff
@@ -1286,7 +1405,7 @@ _curses_window_box_impl(PyCursesWindowObject *self, int group_right_1,
             return NULL;
         }
     }
-    box(self->win,ch1,ch2);
+    (void)box(self->win,ch1,ch2);
```

Comment:
IMO, if a compiler warns here, it does have a point.
```suggestion
    box(self->win,ch1,ch2);
```

Reply:
I'll anyway change it in the next PR where I handle the error myself but I can reduce the diff in this PR.